hack/awsprivatelink/vpc.cf.yaml

```diff
@@ -0,0 +1,265 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
I'm having to take this file 100% on faith.

Reply:
it's a pretty standard VPC setup cloudformation for openshift used by QE